priv/schema/rabbitmq_mqtt.schema

```diff
@@ -88,6 +88,12 @@ end}.
 {mapping, "mqtt.prefetch", "rabbitmq_mqtt.prefetch",
     [{datatype, integer}]}.
 
+%% Enable "Sparkplug B" namespace recognition so that the dot in the
+%% namespace is not translated to a slash
+%%
+%% {sparkplug_b, true},
+{mapping, "mqtt.sparkplug_b", "rabbitmq_mqtt.sparkplug_b",
```

Comment:
@lukebakken Should we use only `sparkplug` for the key and to designate this feature? The translation is done for A and B actually.

Reply:
I just made the change and am re-running tests. Thanks!